engine/src/main/java/org/terasology/rendering/ShaderManagerLwjgl.java

```diff
@@ -92,28 +87,27 @@ public ShaderManagerLwjgl() {
 
     @Override
     public void initShaders() {
-        defaultShaderProgram = prepareAndStoreShaderProgramInstance("default", new ShaderParametersEmpty());
-        defaultTexturedShaderProgram = prepareAndStoreShaderProgramInstance("defaultTextured", new ShaderParametersEmpty());
+        defaultShaderProgram = prepareAndStoreShaderProgramInstance("default");
+        defaultTexturedShaderProgram = prepareAndStoreShaderProgramInstance("defaultTextured");
 
         // TODO: Find a better way to do this
-        prepareAndStoreShaderProgramInstance("post", new ShaderParametersPost());
-        prepareAndStoreShaderProgramInstance("ssao", new ShaderParametersSSAO());
-        prepareAndStoreShaderProgramInstance("lightShafts", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("sobel", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("initialPost", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("prePostComposite", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("highPass", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("blur", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("downSampler", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("toneMapping", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("sky", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("chunk", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("particle", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("shadowMap", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("debug", new ShaderParametersDebug());
-        prepareAndStoreShaderProgramInstance("lightBufferPass", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("lightGeometryPass", new ShaderParametersEmpty());
-        prepareAndStoreShaderProgramInstance("ssaoBlur", new ShaderParametersEmpty());
+        prepareAndStoreShaderProgramInstance("post");
```

Comment:
I'm going with `loadShaderProgram` for now, since "adding where?" is the first question that comes to my mind, when I see `addShaderProgram`